### PR TITLE
docs: rewrite confusing sentences

### PR DESCRIPTION
**Pull request by HonkingGoose**
## Changes:

- Better explain what prefix Renovate bot uses by default if it finds Angular style commits
- Explain that when extending from `config:base` you get `chore` by default, but `fix` for npm dependencies

## Context:

This should prevent the confusion we saw in discussion #10258.

## Documentation (please check one with an [x])

- [x] I have updated the documentation, or
- [ ] No documentation update is required

## How I've tested my work (please tick one)

I have verified these changes via:

- [x] Code inspection only, or
- [ ] Newly added unit tests, or
- [ ] No new tests but ran on a real repository, or
- [ ] Both unit tests + ran on a real repository

### CODE REVIEW

File: docs/usage/semantic-commits.md [hunk 1]
```diff
@@ -15,7 +15,8 @@ When Renovate finds Angular-style commits, Renovate will create commit messages
 - chore(deps): update eslint to v4.2.0
 - fix(deps): update express to v4.16.2
 
-Renovate uses `chore` by default, but uses `fix` for updates to your production dependencies in your `package.json` file.
+Renovate uses the `chore` prefix by default.
+If you extend `config:base`, Renovate still defaults to `chore`, but will use the `fix` prefix for dependencies.
```
Comment (viceice) [inline docs/usage/semantic-commits.md:19]: I think the fix will only applied to npm package dependencies. 🤔

Comment (rarkins) [reply docs/usage/semantic-commits.md:19]: npm and composer I believe

Comment (rarkins) [reply docs/usage/semantic-commits.md:19]: And by `dependencies` we mean production dependencies and not `devDependencies`

Comment (HonkingGoose) [reply docs/usage/semantic-commits.md:19]:
```suggestion
When you extend `config:base`, Renovate still defaults to `chore`, but will use the `fix` prefix for npm production dependencies (`devDependencies` still use `chore`).
```

How about this sentence?

**Pull request merged**